designs/namenode-HA.md

```diff
@@ -0,0 +1,167 @@
+# Namenode HA for HDFS on K8s
```

Comment:
skipping assuming this is directly from the design doc that was reviewed earlier.